### ISSUE TITLE
Not able to parse facebook cookie from request.headers

### ISSUE THREAD

**Issue by narup** [DracoBlue/node-facebook-client#11]
Hi There,

var facebook_cookie_raw = request_headers["cookie"].match(/fbs_[\d]+\=\"([^; ]+)\"/); is returning null in FacebookClient.js for the valid facebook cookie 

my req.headers.cookie has the format 
'"fbsr_116977245018363=MmGgHKw9HEk9U28uapJ0plycsXtKFVfh1JNQi4v_sBo.eyJhbGdvcml0aG0... (length: 276)"

which i am assuming is the right value, because i am logged in to facebook but because above matching returns null.. facebook client doesn't gives me the session back.

Any idea?

Thanks in advance ;)

**Comment by DracoBlue** [comment]
Hi,

how do you manage/make the login happen? Facebook connect?

Do you have this?

```
FB.init({appId: "yourid", logging:false, status: true, cookie: true, xfbml: true});
```

I think cookie is necessary to work.

Hope that helps!

**Comment by narup** [comment]
i do have the cookie and also using FB.init with cookie value true.

**Comment by DracoBlue** [comment]
Can you send me a link to the place where this happens?

**Comment by icarreon** [comment]
I have the same problem, anyone solved it?

**Comment by DracoBlue** [comment]
Can you send me a link to the place where this happens? Which cookies do arrive?

**Comment by liuliu** [comment]
this is the new signed request cookie, see the pull request for more information: https://github.com/DracoBlue/node-facebook-client/pull/16

**Comment by DracoBlue** [comment]
Is fixed in 1.5.0, see 328b6e28d4fb711de599b09ea9c9630c782aa72a
